x-pack/plugins/lens/public/visualizations/gauge/suggestions.ts

```diff
@@ -76,20 +74,55 @@ export const getSuggestions: Visualization<GaugeVisualizationState>['getSuggesti
     ? [
         {
           ...baseSuggestion,
-          previewIcon:
-            state?.shape === GaugeShapes.VERTICAL_BULLET
-              ? IconChartHorizontalBullet
-              : IconChartVerticalBullet,
+          title: gaugeTitlesByType.verticalBullet,
+          previewIcon: IconChartVerticalBullet,
           state: {
             ...baseSuggestion.state,
             ...state,
-            shape:
-              state?.shape === GaugeShapes.VERTICAL_BULLET
-                ? GaugeShapes.HORIZONTAL_BULLET
-                : GaugeShapes.VERTICAL_BULLET,
+            shape: GaugeShapes.VERTICAL_BULLET,
+          },
+        },
+        {
+          ...baseSuggestion,
+          title: gaugeTitlesByType.horizontalBullet,
+          previewIcon: IconChartHorizontalBullet,
+          state: {
+            ...baseSuggestion.state,
+            ...state,
+            shape: GaugeShapes.HORIZONTAL_BULLET,
+          },
+        },
+        {
+          ...baseSuggestion,
+          title: gaugeTitlesByType.arc,
+          previewIcon: IconChartGaugeArc,
+          state: {
+            ...baseSuggestion.state,
+            ...state,
+            shape: GaugeShapes.ARC,
+          },
+        },
+        {
+          ...baseSuggestion,
+          title: gaugeTitlesByType.majorArc,
+          previewIcon: IconChartGaugeMajorArc,
+          state: {
+            ...baseSuggestion.state,
+            ...state,
+            shape: GaugeShapes.MAJOR_ARC,
+          },
+        },
+        {
+          ...baseSuggestion,
+          title: gaugeTitlesByType.circle,
+          previewIcon: IconChartGaugeCircle,
+          state: {
+            ...baseSuggestion.state,
+            ...state,
+            shape: GaugeShapes.CIRCLE,
```

Comment:
should we avoid suggesting arcs? I generally believe that linear gauges are better suited for most cases, arcs are just ok for old aesthetic principles

Reply:
I like MarcoV thinking but I also know people will use them anyway, they are just too pretty 😁  🙈 . But anyway, let's generate suggestions for all so the chart switch picks them up, but set `hide: true` for the ones we want to hide so we don't see them in suggestions panel.

Reply:
What if we just set the scores more favorably to the bullets? I think there is still merit in the `Major Arc` but at a lower level. Say score like this...

|Type|score|
|:-|:-:|
|Bullet vertical|`1`|
|Bullet horizontal|`1`|
|Gauge arc|`0.1`|
|Gauge major arc|`0.5`|
|Gauge circle|`0.1`|

Reply:
Also this is fine to me, I'm not sure how the score works here, but this is definitely ok

Reply:
Updated the `scores` in 19ec260ceac089bf0abe94dfa4db2f5417d52fc5 to reflect the priority shown above. With this, `Gauge arc` and `Gauge circle` will show last within the gauge grouping thus after all other gauges and metrics like below.

<img width="969" alt="image" src="https://github.com/elastic/kibana/assets/19007109/a2f915c0-21e3-4c93-ab38-697099328867">